hl7/4.2/modules/ROOT/edits/hl7-connector-config-topics.adoc

```diff
@@ -0,0 +1,269 @@
+
+Here's an example:
+
+[source,yaml,linenums]
+----
+A01: { XYZ: ADT_A01, ACK: ACK }
+A02: { XYZ: ADT_A02, ACK: ACK }
+A03: { XYZ: ADT_A03, ACK: ACK }
+A04: { XYZ: ADT_A01, ACK: ACK }
+A05: { XYZ: ADT_A05, ACK: ACK }
+A06: { XYZ: ADT_A06, ACK: ACK }
+A07: { XYZ: ADT_A06, ACK: ACK }
+A08: { XYZ: ADT_A01, ACK: ACK }
+----
+
+Each version of HL7 defines a different set of mappings from the event type and
+message code to the message structure. The default mappings are provided in the
+same JAR as the standard HL7 schema definitions, in files named `event-message.yaml`.
+You use the same type of paths for these mapping definitions as for the actual message structure schemas.
+
+=== General Tab
+
+Use the *General* tab to configure settings for reading and writing HL7 messages:
+
+* HL7 character encoding, always used for writing messages and used
+when reading messages unless a different encoding is specified by MSH-18 (Character Set).
+* Disable numeric prefixes for data keys - this option is true by
+default, which turns off numeric prefixes for segment data. The
+only reason to turn this option off is for compatibility with mappings
+defined for the HL7 EDI Connector 3.0.0.
+* Manually create or edit the list of schemas.
+
+=== Set Your HL7 Identification in the Visual Editor
+
+You can configure the Message Header (MSH) application and facility
+identification for you and your trading partner on the HL7 EDI connector configuration.
+
+The values you set are used when writing HL7 messages to supply the
+namespace ID, universal ID, and universal ID type, and are verified
+in receive messages. If you don't want to restrict incoming messages
+then leave these blank, and set the values for outgoing messages
+on the write operation or the actual outgoing message. Values set on
+the write operation override the connector configuration, and values
+set directly on the message override both the connector configuration
+and any values set on the write operation.
+
+In Studio, these values are set in these Global Element Properties.
+
+* Self identification parameters identify your side of the trading partner relationship.
++
+Self identification settings:
++
+[source,text,linenums]
+----
+Mule Application Namespace ID (MSH-03/HD-01 when sending, MSH-05/HD-01 when receiving)
+Mule Application Universal ID (MSH-03/HD-02 when sending, MSH-05/HD-02 when receiving)
+Mule Application Universal ID Type (MSH-03/HD-03 when sending, MSH-05/HD-03 when receiving)
+----
++
+* Partner identification parameters identify your trading partner.
++
+Partner identification settings:
++
+[source,text,linenums]
+----
+Partner Application Namespace ID (MSH-03/HD-01 when receiving, MSH-05/HD-01 when sending)
+Partner Application Universal ID (MSH-03/HD-02 when receiving, MSH-05/HD-02 when sending)
+Partner Application Universal ID Type (MSH-03/HD-03 when receiving, MSH-05/HD-03 when sending)
+----
+
+=== Parser Tab
+
+You can set the following options if needed:
```

Comment:
Do you want to say anything more about "if needed"? Under what circumstances might they be needed?

Reply:
They are optional, that should be enough as users will know when then need to use this.